src/__integration_tests__/paymentflow.integration.test.ts

```diff
@@ -301,6 +314,72 @@ describe("Checkout fails to calculate fee", () => {
 
   });
 
+  it.each([
+    ["it", itTranslation],
+    ["en", enTranslation],
+    ["fr", frTranslation],
+    ["de", deTranslation],
+    ["sl", slTranslation]
+  ])("Should fails calculate fee with 404 for CARDS and language [%s]", async (lang, translation) => {
+    /*
+     * Calculate fee fails
+    */
+    selectLanguage(lang);
+
+    await fillAndSubmitCardDataForm(PSP_NOT_FOUND_FAIL, VALID_FISCAL_CODE, EMAIL, VALID_CARD_DATA);
+    
+    const pspNotFoundTitleId = "#pspNotFoundTitleId";
+    const pspNotFoundTitleElem = await page.waitForSelector(
+      pspNotFoundTitleId
+    );
+    const pspNotFoundTitleText = await pspNotFoundTitleElem.evaluate((el) => el.textContent);
+    expect(pspNotFoundTitleText).toContain(translation.pspUnavailable.title);
+
+    const pspNotFoundCtaId = "#pspNotFoundCtaId";
+    const pspNotFoundCtaElem = await page.waitForSelector(pspNotFoundCtaId);
+    const pspNotFoundCtaText = await pspNotFoundCtaElem.evaluate((el) => el.textContent);
+    expect(pspNotFoundCtaText).toContain(translation.pspUnavailable.cta.primary);
+
+    const errorDescriptionId = "#pspNotFoundBodyId";
+    const errorDescriptionElem = await page.waitForSelector(errorDescriptionId);
+    const errorDescriptionText = await errorDescriptionElem.evaluate((el) => el.textContent);
+    expect(errorDescriptionText).toContain(translation.pspUnavailable.body);
+
+  });
+
+  it.each([
+    ["it", itTranslation],
+    ["en", enTranslation],
+    ["fr", frTranslation],
+    ["de", deTranslation],
+    ["sl", slTranslation]
+  ])("Should fails calculate fee with 404 for SATISPAY and language [%s]", async (lang, translation) => {
+    /*
+     * Calculate fee fails
+    */
+    selectLanguage(lang);
+
+    await fillAndSubmitSatispayPayment(PSP_NOT_FOUND_FAIL, VALID_FISCAL_CODE, EMAIL);
+
+    const pspNotFoundTitleId = "#pspNotFoundTitleId";
+    const pspNotFoundTitleElem = await page.waitForSelector(
+      pspNotFoundTitleId
+    );
+    const pspNotFoundTitleText = await pspNotFoundTitleElem.evaluate((el) => el.textContent);
+    expect(pspNotFoundTitleText).toContain(translation.pspUnavailable.title);
+
+    const pspNotFoundCtaId = "#pspNotFoundCtaId";
+    const pspNotFoundCtaElem = await page.waitForSelector(pspNotFoundCtaId);
+    const pspNotFoundCtaText = await pspNotFoundCtaElem.evaluate((el) => el.textContent);
+    expect(pspNotFoundCtaText).toContain(translation.pspUnavailable.cta.primary);
+
+    const errorDescriptionId = "#pspNotFoundBodyId";
+    const errorDescriptionElem = await page.waitForSelector(errorDescriptionId);
+    const errorDescriptionText = await errorDescriptionElem.evaluate((el) => el.textContent);
+    expect(errorDescriptionText).toContain(translation.pspUnavailable.body);
+
```

Comment:
Same as above, even if here the modal is shown on the payment choice page, so you should check if the page is the same.

Reply:
done